src/dotnet/commands/dotnet-list/dotnet-list-p2ps/Program.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Tools.List.ProjectToProjectReferences
+{
+    public class ListProjectToProjectReferencesCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false)
+            {
+                Name = "dotnet list p2ps",
```

Comment:
I'll change it but I'm pretty sure I have copy pasted this from some other command and changed similarly which would mean that we have something like that in other places - I'll take a look if that's the case.